mindsdb/utilities/config.py

```diff
@@ -306,6 +320,35 @@ def prepare_env_config(self) -> None:
             self._env_config['logging']['handlers']['file']['enabled'] = True
         # endregion
 
+        # region server type
+        server_type = os.environ.get('MINDSDB_HTTP_SERVER_TYPE', '').lower()
```

Comment:
Should we catch scenarios where server type is invalid, outside of supported values?

Reply:
added to `raise_warnings`